docs/src/main/sphinx/sql/select.rst

```diff
@@ -1147,6 +1147,53 @@ from relations on the left side of the join::
      Mary    |     9
     (6 rows)
 
+Another example of ``UNNEST`` referencing columns from relation on the left side of join::
```

Comment:
Isn't this covered by 

https://github.com/trinodb/trino/blob/565375bb7ca6822384b3f81e0d5fccb7df6ef085/docs/src/main/sphinx/sql/select.rst?plain=1#L1127-L1148 

?

Reply:
Above changes are done based on https://github.com/trinodb/trino/issues/17897#issuecomment-1646651732